### ISSUE TITLE
[Bug] Ryujinx UI disappears after exiting full screen (macOS)

### ISSUE THREAD

**Issue by morceaudebois** [Ryujinx/Ryujinx#4960]
### Description of the issue

When exiting full screen using the native green button of macOS, the top and bottom bar are absent from the Ryujinx window. They reappear if you hit full screen again, then the escape key.

### Ryujinx after exiting full screen with green button :
<img width="1392" alt="Screenshot 2023-05-15 at 11 57 16" src="https://github.com/Ryujinx/Ryujinx/assets/50768582/ee080663-ddaf-44dc-8549-e27e99689c2a">

### Ryujinx after exiting full screen with escape key :
<img width="1392" alt="Screenshot 2023-05-15 at 11 57 11" src="https://github.com/Ryujinx/Ryujinx/assets/50768582/5fbacf16-7e6c-4518-ab13-23348e53fa21">

### Reproduction steps

1. Open a game in full screen
2. Exit using the green macOS button 

### Log file

*

### OS

macOS 12

### Ryujinx version

1.1.799

**Comment by ralph303** [comment]
I can confirm this, altough it's a rather cosmetic bug with an easy workaround ;)

**Comment by morceaudebois** [comment]
Yup! As long as you know the workaround it's fine, but I was pretty frustrated for a while before I found out about it 😅
